drv/transceivers-server/src/udp.rs

```diff
@@ -582,14 +608,37 @@ impl ServerImpl {
                     num_err_bytes,
                 )
             }
+            HostRequest::ClearDisableLatch(modules) => {
+                ringbuf_entry!(Trace::ClearDisableLatch(modules));
+                let mask = LogicalPortMask::from(modules);
+                self.disabled &= !mask;
+                // This operation just sets internal SP state, so it is always
+                // successful. However, invalid modules may been specified by
+                // the host so we need to check that anyway.
+                let result =
+                    ModuleResultNoFailure::new(mask, LogicalPortMask(0))
+                        .unwrap();
+                let success = ModuleId::from(result.success());
+                let (num_err_bytes, failed_modules) =
+                    self.handle_errors(modules, result, out);
+                (
+                    MessageBody::SpResponse(SpResponse::Ack {
+                        modules: success.into(),
```

Comment:
I think the `.into()` is unnecessary here, `success` is already a `ModuleId`.

Reply:
Yup, done in 0bcc3e8bf42a13318e35b8d6612dd44471157cec